src/agent/protocols/protos/agent.proto

```diff
@@ -357,10 +357,6 @@ message AgentDetails {
 
 	// List of available storage handlers.
 	repeated string storage_handlers = 4;
-
-	// Set only if the agent is built with seccomp support and the guest
-	// environment supports seccomp.
-	bool supports_seccomp = 5;
```

Comment:
Do we want to update the agent API?

Reply:
Yes.
I think that  `supports_seccomp` flag always will be `true`, so the flag is not used anymore.
In addition, we can control whether the agent's seccomp support is enabled when building the agent.
If users want to control the flag through the agent API, we should leave it, but I'm not sure if there is such a use case.
Could you give me your thoughts?

Reply:
If we don't use a field, we should leave it here for rpc protocol compatibility. Please don't remove it.

And in fact, the field is useful on the runtime side so that we can dynamically determine if seccomp info should be passed to the agent or not. Please look at `src/runtime/virtcontainers/kata_agent.go` and see how it works currently.